src/DistributedMetadata/CatalogService.h

```diff
@@ -62,6 +62,7 @@ class CatalogService final : public MetadataService
 
     std::pair<TablePtr, StoragePtr> findTableStorageById(const UUID & uuid) const;
     std::pair<TablePtr, StoragePtr> findTableStorageByName(const String & database, const String & table) const;
+    inline void deleteTableStorageByName(const String & database, const String & table);
```

Comment:
`inline` shall be specified along with the implementation instead of function declaration which means let's remove the inline keyword here to avoid confusion.

Reply:
fixed